corehq/apps/app_manager/suite_xml/sections/details.py

```diff
@@ -235,7 +235,9 @@ def build_detail(self, module, detail_type, detail, detail_column_infos, tabs=No
 
                 if module_offers_search(module) and not module_uses_inline_search(module):
                     in_search = module_loads_registry_case(module) or "search" in id
-                    d.actions.append(self._get_case_search_action(module, in_search=in_search))
+                    # don't add search again action in split screen
```

Comment:
Heads up that https://github.com/dimagi/commcare-hq/pull/32974 is going to conflict/affect this. Can you merge master when you have a chance?

Reply:
nevermind, you were merging while I was reviewing

Reply:
Looking at the new diff, should [this new call to get_case_search_action](https://github.com/dimagi/commcare-hq/blob/b616afe421992ccc01293154d7e8a75e34eeb137/corehq/apps/app_manager/suite_xml/sections/details.py#L687-L694) also check the split screen flag?

Reply:
I saw that and wasn't immediately sure, but yes I think it should.